pylearn2/models/tests/test_costs.py

```diff
@@ -0,0 +1,131 @@
+"""
+Note: Cost functions are not implemented for RectifierConvNonlinearity,
+TanhConvNonlinearity, RectifiedLinear, and Tanh.  Verify that the implemented 
+cost functions for convolutional layers give the correct output
+by comparing to standard MLP's.
+"""
+
+import numpy as np
+import theano
+import theano.tensor as T
+from pylearn2.models.mlp import MLP
+from pylearn2.models.mlp import Sigmoid, Tanh, Linear, RectifiedLinear
+from pylearn2.models.mlp import ConvElemwise
+from pylearn2.space import Conv2DSpace
+from pylearn2.models.mlp import SigmoidConvNonlinearity
+from pylearn2.models.mlp import TanhConvNonlinearity
+from pylearn2.models.mlp import IdentityConvNonlinearity
+from pylearn2.models.mlp import  RectifierConvNonlinearity
+
+def test_implemented_case(ConvNonlinearity, MLPNonlinearity):
```

Comment:
Since this function is not supposed to be executed directly by nosetests, it should not start with `test_`.
Could you rename it `check_implemented_case` for instance?